internal/controllers/topology/cluster/patches/inline/json_patch_generator_test.go

```diff
@@ -1786,23 +2127,46 @@ owner: root:root
 			},
 			want: &apiextensionsv1.JSON{Raw: []byte(`"A1"`)},
 		},
-		// Pick up config for a specific MD Class
+		// Pick up config for a specific MD/MP Class
 		{
-			name:     "Should render a object property with a lookup based on a builtin variable (class)",
-			template: `{{ (index .mdConfig .builtin.machineDeployment.class).config }}`,
+			name: "Should render a object property with a lookup based on a builtin variable (class)",
+			// TODO: We should probably document more clearly that the output of the go template will be passed into
```

Comment:
Probably a `TODO: delete me`. Just a note for myself to not forget. But donno what the best place for this is :thinking: